src/ui/components/waiting.c

```diff
@@ -21,13 +22,12 @@
 #include <screen.h>
 #include <ui/ui_util.h>
 
+#include <stdbool.h>
 #include <string.h>
 
-static void _render(component_t* component)
-{
-    // TODO - add an interesting animation?
-    ui_util_component_render_subcomponents(component);
-}
+typedef struct {
+    bool host_connected;
```

Comment:
based on other comments I think the name of this variable is wrong. Is this truly `true` if and only if the host is connected?